prowler/providers/aws/services/organizations/organizations_scp_check_deny_regions/organizations_scp_check_deny_regions.py

```diff
@@ -29,69 +29,76 @@ def execute(self):
                     report.status_extended = f"AWS Organizations {organizations_client.organization.id} does not have SCP policies."
                     # We use this flag if we find a statement that is restricting regions but not all the configured ones:
                     is_region_restricted_statement = False
+                    has_statements = False
 
                     for policy in organizations_client.organization.policies.get(
                         "SERVICE_CONTROL_POLICY", []
                     ):
                         # Statements are not always list
                         statements = policy.content.get("Statement")
```

Comment:
@danibarranqueroo I think it is enough by only changing this line
```suggestion
                        statements = policy.content.get("Statement",[])
```

Reply:
It was needed to also change this line:

```
if type(policy.content["Statement"]) is not list:
                            statements = [policy.content.get("Statement")]
```

Because `policy.content["Statement"]` is not accessible if there is no Statement, but following your approach is easier the solution. Thanks for the suggestion! 🚀